### ISSUE TITLE
java.lang.NullPointerException: Cannot invoke "com.intellij.openapi.wm.StatusBar.updateWidget(String)" because "this.widget.statusBar" is null

### ISSUE THREAD

**Issue by CrazyBoy49z** [wakatime/jetbrains-wakatime#251]
```
java.lang.NullPointerException: Cannot invoke "com.intellij.openapi.wm.StatusBar.updateWidget(String)" because "this.widget.statusBar" is null
	at com.wakatime.intellij.plugin.CustomStatusBar$WakaTimeStatusBarWidget$StatusBarPresenter.getPopupStep(CustomStatusBar.java:98)
	at com.intellij.openapi.wm.StatusBarWidget$MultipleTextValuesPresentation.getPopup(StatusBarWidget.kt:70)
	at com.intellij.openapi.wm.impl.status.MultipleTextValues$2.onClick(IdeStatusBarImpl.kt:867)
	at com.intellij.ui.ClickListener$1.mouseReleased(ClickListener.java:58)
	at java.desktop/java.awt.Component.processMouseEvent(Component.java:6656)
	at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3385)
	at java.desktop/java.awt.Component.processEvent(Component.java:6421)
	at java.desktop/java.awt.Container.processEvent(Container.java:2266)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5026)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2324)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4854)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4948)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4575)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4516)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2310)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2804)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4854)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:790)
	at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:739)
	at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:731)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:97)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:763)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:761)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:760)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:666)
	at com.intellij.ide.IdeEventQueue.dispatchMouseEvent(IdeEventQueue.kt:614)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:569)
	at com.intellij.ide.IdeEventQueue.access$_dispatchEvent(IdeEventQueue.kt:68)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:349)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:348)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:787)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:348)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:343)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$1(IdeEventQueue.kt:994)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:113)
	at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:994)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$4(IdeEventQueue.kt:343)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:831)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:385)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)

```

```
PhpStorm 2023.1
Build #PS-231.8109.199, built on March 30, 2023
Licensed to Yurij Finiv
Subscription is active until September 17, 2023.
Runtime version: 17.0.6+10-b829.5 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 12.6.4
GC: G1 Young Generation, G1 Old Generation
Memory: 4048M
Cores: 8
Metal Rendering is ON
Registry:
    debugger.new.tool.window.layout=true
    ide.windowSystem.autoShowProcessPopup=true
    run.processes.with.pty=TRUE
    ide.experimental.ui=true
    ide.balloon.shadow.size=0

Non-Bundled Plugins:
    com.github.czy211.licensetemplate (2.0.0)
    LiveTemplatePreview (0.3)
    ideanginx9 (0.1.9)
    color.scheme.Darcula Red (0.3)
    com.intellij.javafx (1.0.4)
    com.guohanlin.JsonToAnyLanguage (1.1.0)
    lermitage.intellij.extratci (1.24.0)
    MultiHighlight (3.0.0)
    cc.moky.intellij.plugin.svga-support (1.0.9)
    io.aesy.regex101 (0.2)
    com.github.warningimhack3r.npmupdatedependencies (1.1.2)
    me.x150.intellij-code-screenshots (1.4.1)
    com.github.copilot (1.2.5.2529-nightly)
    com.materkey.codepoints (1.0.2)
    com.mroche.JsonHelper (1.2.0)
    dev.jamiecraane.gptmentorplugin (0.0.7)
    lermitage.intellij.iconviewer (1.20.0)
    com.intellij.properties (231.8109.91)
    lermitage.intellij.worldclock (1.6.0)
    com.wix.sasslint (1.1.2)
    bundled-phpstorm-help (231.8109.199)
    String Manipulation (9.7.1)
    krasa.CpuUsageIndicator (1.15)
    Key Promoter X (2023.1.0)
    com.xtu.plugins.reviewer (1.1.1)
    ee.carlrobert.chatgpt (1.10.0)
    com.wakatime.intellij.plugin (14.1.4)
    com.xiaobing1024.laravel (1.1.1)
    de.achimonline.github_markdown_emojis (1.2.2)
    ski.chrzanow.foldableprojectview (1.1.5)
    com.chriscarini.jetbrains.jetbrains-auto-power-saver (3.0.0)
    lermitage.intellij.battery.status (1.26)
    io.snyk.snyk-intellij-plugin (2.4.55)
    com.github.inxilpro.intellijalpine (v0.4.2)
    com.github.blarc.ai-commits-intellij-plugin (0.6.1)
    com.mallowigi (90.0.0)
    zielu.gittoolbox (500.0.6+213)
    mobi.hsz.idea.gitignore (4.5.0)
    izhangzhihao.rainbow.brackets (2023.2.6)
    com.alicannklc.laravel.snippets (1.1.1)
    com.kalessil.phpStorm.phpInspectionsEA (4.0.7.1)
    com.yalhyane.intellij.phpaicode.php-ai-code (1.0.1)
    com.funivan.idea.phpClean (2023.04.01)
    cn.yiiguxing.plugin.translate (3.4.2)
    ru.adelf.idea.dotenv (2023.1)
    com.mallowigi.colorHighlighter (15.0.0)
    de.espend.idea.php.annotation (9.2.0)
    fr.adrienbrault.idea.symfony2plugin (2022.1.246)
    com.github.dinbtechit.vscodetheme (1.7.8)
    net.rentalhost.plugins.php.hammer (0.21.0)
    dev.nybroe.collector (0.4.0)
    com.laravel_idea.plugin (7.0.2.231)
    com.pestphp.pest-intellij (1.9.2)
    com.intellij.microservices.ui (231.8109.93)
```

**Comment by finezhaooo** [comment]
same issue when i enabled new UI in intellijIDEA,and connected to WSL.

more info:

IntelliJ IDEA 2023.1 (Ultimate Edition)
Build #IU-231.8109.175, built on March 28, 2023
Runtime version: 17.0.6+10-b829.5 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 11.0
GC: G1 Young Generation, G1 Old Generation
Memory: 750M
Cores: 12
Registry:
    debugger.new.tool.window.layout=true
    ide.experimental.ui=true


Kotlin: 231-1.8.20-IJ8109.175

**Comment by fzzio** [comment]
Same for me at the community edition of IntelliJ in Ubuntu 22.04
```
java.lang.NullPointerException: Cannot invoke "com.intellij.openapi.wm.StatusBar.updateWidget(String)" because "this.widget.statusBar" is null
	at com.wakatime.intellij.plugin.CustomStatusBar$WakaTimeStatusBarWidget$StatusBarPresenter.getPopupStep(CustomStatusBar.java:98)
	at com.intellij.openapi.wm.StatusBarWidget$MultipleTextValuesPresentation.getPopup(StatusBarWidget.kt:70)
	at com.intellij.openapi.wm.impl.status.MultipleTextValues$2.onClick(IdeStatusBarImpl.kt:868)
	at com.intellij.ui.ClickListener$1.mouseReleased(ClickListener.java:58)
	at java.desktop/java.awt.Component.processMouseEvent(Component.java:6656)
	at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3385)
	at java.desktop/java.awt.Component.processEvent(Component.java:6421)
	at java.desktop/java.awt.Container.processEvent(Container.java:2266)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5026)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2324)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4854)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4948)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4575)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4516)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2310)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2804)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4854)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:790)
	at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:739)
	at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:731)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:97)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:763)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:761)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:760)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:667)
	at com.intellij.ide.IdeEventQueue.dispatchMouseEvent(IdeEventQueue.kt:615)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:570)
	at com.intellij.ide.IdeEventQueue.access$_dispatchEvent(IdeEventQueue.kt:68)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:349)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:348)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:787)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:348)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:343)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$1(IdeEventQueue.kt:995)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:113)
	at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:995)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$4(IdeEventQueue.kt:343)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:831)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:385)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)

```

**Comment by alanhamlett** [comment]
Duplicate of #257, fixed in v14.1.5.
